regression-test/data/ddl_p0/test_ctas.out

```diff
@@ -1,6 +1,6 @@
 -- This file is automatically generated. You should know what you did if you want to edit this
 -- !select --
-test_ctas1	CREATE TABLE `test_ctas1` (\n  `test_varchar` varchar(150) NULL,\n  `test_datetime` datetime NULL,\n  `test_default_timestamp` datetime NULL DEFAULT CURRENT_TIMESTAMP\n) ENGINE=OLAP\nDUPLICATE KEY(`test_varchar`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`test_varchar`) BUCKETS 10\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
+test_ctas1	CREATE TABLE IF NOT EXISTS `test_ctas1` (\n  `test_varchar` varchar(150) NULL,\n  `test_datetime` datetime NULL,\n  `test_default_timestamp` datetime NULL DEFAULT CURRENT_TIMESTAMP\n) ENGINE=OLAP\nDUPLICATE KEY(`test_varchar`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`test_varchar`) BUCKETS 10\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
```

Comment:
even create table with “IF NOT EXISTS”，which will not be displayed when show create table， u should move “IF NOT EXISTS”